### PR TITLE
node: add errorMonitor on EventEmitter

### PR DESCRIPTION
**Pull request by Flarna**
The property `errorMonitor` is a static property on class `EventEmitter`.
This avoid the need to import `errorMonitor` separate of `EventEmitter`.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/dist/latest-v14.x/docs/api/events.html#events_eventemitter_errormonitor
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
thx for update!

**Review by nguymin4 — APPROVED** [review]

**Review by a-tarasyuk — APPROVED** [review]

**Comment by Flarna** [comment]
@typescript-bot Why is this PR still in "Waiting for Code Reviews" state?

**Review by r3nya — APPROVED** [review]

**Comment by Flarna** [comment]
@sandersn I have the feeling that @typescript-bot is lazy in updating the status of PRs. For example this one was moved from `Waiting for Code Reviews` to `Needs Maintainer Review` ~4 days after approval.
Similar #44769 seems to be stuck in the `Waiting for Code Reviews` state.

**Review by addaleax — APPROVED** [review]

**Pull request merged**
